### PR TITLE
User's identity is now recorded by default

### PR DESCRIPTION
**Pull request by bkuhl**

### CODE REVIEW

**Comment by bryannielsen** [comment]
It would be great to see this merged in!

File: src/RollbarServiceProvider.php [hunk 1]
```diff
@@ -23,6 +25,18 @@ public function boot()
 
         // Listen to log messages.
         $app['log']->listen(function ($level, $message, $context) use ($app) {
+            
+            $user = Auth::user();
+
+            // log logged in user details
+            if ($user instanceof User) {
```
Comment (bram1028) [inline src/RollbarServiceProvider.php]: Can't assume \App\User, should use \Illuminate\Contracts\Auth\Authenticatable.

Comment (bkuhl) [inline src/RollbarServiceProvider.php]: Good point, I've updated this to use `Authenticatable` and optionally try to append other attributes not available on that contract

**Comment by mbardelmeijer** [comment]
Yes please! 👍

**Comment by bkuhl** [comment]
This has been updated with the base branch and merge conflicts resolved.

**Comment by bkuhl** [comment]
I see coverage and some failing tests need to be addressed.  I'll close this and resubmit soon.

**Comment by bkuhl** [comment]
Nevermind... adding tests for these closures is painful.  I'll leave it up to the authors to add this support.

**Pull request closed without merging**